src/coreclr/jit/gentree.h

```diff
@@ -3250,6 +3254,194 @@ struct GenTreeStrCon : public GenTree
 #endif
 };
 
+// GenTreeVecCon -- vector  constant (GT_CNS_VEC)
+//
+struct GenTreeVecCon : public GenTree
+{
+    union {
+        simd8_t  gtSimd8Val;
+        simd12_t gtSimd12Val;
+        simd16_t gtSimd16Val;
+        simd32_t gtSimd32Val;
+    };
+
+private:
+    unsigned char gtSimdBaseJitType; // SIMD vector base JIT type
```

Comment:
In that case it should have an appropriate `TODO`, indicating this is a workaround that is no longer needed.

Reply:
Fixed. But more generally I believe such TODOs in source are problematic.

The 1st-class-struct work is expansive and a large portion of locations in the JIT do not have comments or do not use a consistent prefix to allow identifying the impacted locations.

This type of code isn't logic that is required to be fixed to remove handles and is something that should naturally fall out of removal once the handle cache is removed. Otherwise, its likely that comment will simply get out of sync over time and cause more confusion in the long term.